tests/test_computetools.py

```diff
@@ -0,0 +1,28 @@
+"Test for the BorisPush Computetool in turbopy/computetools.py"
+
+import numpy as np
+from turbopy.computetools import BorisPush
+from turbopy.core import Simulation
+
+
+def test_init_should_create_object():
+    """Method that creates a BorisPush object to test __init__"""
+    init_example = BorisPush(Simulation({"type": "BorisPush"}),
+                             {"type": "BorisPush"})
+    assert isinstance(init_example, BorisPush)
+    assert init_example.c2 == 2.9979e8 ** 2
+
+
+def test_push():
+    """Tests the functionality of the push method"""
+    input_data = {"Clock": {"start_time": 0, "end_time": 1, "dt": 0.1},
+                  "type": "BorisPush",
+                  "Grid": {"N": 10, "min": 20, "max": 30},
+                  "PhysicsModules": {}}
+    owner = Simulation(input_data)
+    owner.prepare_simulation()
+    push_example = BorisPush(owner, input_data)
+    arr = np.ndarray(shape=(2, 2), dtype=float)
+    arr.fill(1)
+    push_example.push(arr, arr, 1, 1, arr, arr)
+    assert True
```

Comment:
I'd do something like this:
```suggestion
    charge = 1.6022e-19      # proton charge in C
    mass = 1.6726e-27        # proton mass in kg
    E = np.zeros(3)
    B = np.zeros(3)
    x0 = np.array([[0, 0, 0.0]], dtype=np.float)
    v0 = np.array([[0, 0, 3.0e6]], dtype=np.float)  # velocity in m/s
    p0 = mass * v0              # nonrelativistic momentum (v<<c)
    dt = 1e-9                        # 1 ns timestep
    Nt = 10
    xfinal = v0 * Nt * dt
    for i in range(Nt):
        pusher.push(x0, p0, charge, mass, E, B)
    assert np.allclose(x0, xfinal)
```

Reply:
The suggestion above checks that the position gets updated correctly if there are no fields. There should probably also be some checks for cases where the E and B fields are not zero. But we can do those later.